Municipality/kapafunctions.pl

```diff
@@ -0,0 +1 @@
+remove_from_list(List1, Elem, List2) :- delete(List1, Elem, List2).
```

Comment:
This is necessary :wink: , delete is already a Goal term, and prohibited to use otherwise, but we haven't used this function because we had changed the behaviour so this file will be removed